src/mbgl/shaders/fill_outline.cpp

```diff
@@ -41,7 +41,7 @@ void main() {
     
 
     float dist = length(v_pos - gl_FragCoord.xy);
-    float alpha = smoothstep(1.0, 0.0, dist);
+    float alpha = 1.0 - smoothstep(0.0, 1.0, dist);
```

Comment:
See https://github.com/mapbox/mapbox-gl-native/issues/8751 and https://github.com/mapbox/mapbox-gl-native/pull/8760